### PR TITLE
Automatic generation of the helm docs - v0.x

### LINKED ISSUE #221: Generate automatically helm docs
In order not to miss new arguments, it would be interesting to integrate [helm-docs](https://github.com/norwoodj/helm-docs) in the release process.

### PR DESCRIPTION
**Pull request by outscale-mdr**
In order to have helm doc always up to date, here is a PR to use [helm-docs](https://github.com/norwoodj/helm-docs) tools to generate the documentation using the chart directly?

Closes #221

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,7 @@ help:
 	@echo "  - verify             : check code"
 	@echo "  - test               : run all tests"
 	@echo "  - test-e2e-single-az : run e2e tests"
+	@echo "  - helm-docs		  : generate helm doc"
```
Comment (jerome-jutteau) [inline Makefile:49]: spaces :)

File: docs/release.md [hunk 2]
```diff
@@ -5,10 +5,11 @@
...
+5. Tag the release
 ```shell
 git tag -a vX.X.X -m "Release vX.X.X"
 ```
-5. Generate the docker image with `make build-image`
-6. Push the ocker image to the registry
-7. Make the release on Github 
\ No newline at end of file
+6. Generate the docker image with `make build-image`
+7. Push the ocker image to the registry
```
Comment (jerome-jutteau) [inline docs/release.md:14]: typo ocker -> docker

**Review by jerome-jutteau — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (outscale-mdr) [reply Makefile]: Oups

File: docs/release.md (hunk 2 above)
Comment (outscale-mdr) [reply docs/release.md]: thank you

**Pull request merged**